quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusValidator.java

```diff
@@ -130,13 +130,61 @@ private void validatePropertyValue(String propertyName, ExtendedConfigDescriptio
 			return;
 		}
 
-		// TODO validate boolean, int, enums types
-		String type = metadata.getType();
+		String value = property.getValue().getText().trim();
+		if (value.length() == 0) {
+			return;
+		}
+		
+		String errorMessage = null;
+		if (isInvalidEnumValue(metadata, value)) {
+			errorMessage = "Invalid enum value: '" + property.getValue().getText().trim() + "' is invalid for type " + metadata.getType();

```

Comment:
replace with 

`errorMessage = "Invalid enum value: '" + value+ "' is invalid for type " + metadata.getType();`